apps/journeys-admin/src/components/SingleJourney/ShareSection/ShareSection.tsx

```diff
@@ -0,0 +1,204 @@
+import { ReactElement, useState } from 'react'
+import {
+  Box,
+  IconButton,
+  TextField,
+  InputAdornment,
+  Alert,
+  Snackbar,
+  Fade,
+  useTheme,
+  Menu,
+  MenuItem,
+  ListItemIcon,
+  ListItemText,
+  Button,
+  Link,
+  Typography
+} from '@mui/material'
+import InsertLinkIcon from '@mui/icons-material/InsertLink'
+import CheckCircleIcon from '@mui/icons-material/CheckCircle'
+import MoreHorizIcon from '@mui/icons-material/MoreHoriz'
+import ContentCopyIcon from '@mui/icons-material/ContentCopy'
+import EditIcon from '@mui/icons-material/Edit'
+import { useBreakpoints } from '@core/shared/ui'
+
+export interface ShareSectionProps {
+  slug: string
+  forceMenu?: boolean // used only for storybook testing
+}
+
+const ShareSection = ({ slug, forceMenu }: ShareSectionProps): ReactElement => {
```

Comment:
this should be function
```suggestion
export function ShareSection ({ slug, forceMenu }: ShareSectionProps): ReactElement {
```